TYPES.adoc

```diff
@@ -0,0 +1,94 @@
+= Content Types
+
+This is meant to resolve
+link:https://issues.jenkins-ci.org/browse/WEBSITE-21[WEBSITE-21] and define
+some of the content (read: data) types to be used for the `jenkins.io` site.
+
+
+== Timeline
+
+These content types can be thought of as a timeline of related pieces of
+content. Much like a blog or twitter feed, these pieces of content can be
+strung together on a chronological timeline.
+
+
+=== Blog Post
+
+**Template**: post
+
+**Data:**
+
+* `title`
+* `tags`
+* `body`
+
+=== Event
+
+**Template**: event
+
+**Data:**
+
+* `title`
+* `date`
```

Comment:
My definition of date follows the `date(1)` one where time is part of the date output :smiley_cat: 

I think an optional meetup URL and GPS makes sense, ensuring that the GPS is legitimate will be tricky